src/towncrier/test/test_build.py

```diff
@@ -958,123 +893,115 @@ def do_build_once_with(version, fragment_file, fragment):
                 ).lstrip(),
             )
 
-    def test_bullet_points_false(self):
+    @with_project(
+        config="""
+        [tool.towncrier]
+        template="towncrier:single-file-no-bullets"
+        all_bullets=false
+        """
+    )
+    def test_bullet_points_false(self, runner):
         """
         When all_bullets is false, subsequent lines are not indented.
 
         The automatic ticket number inserted by towncrier will align with the
         manual bullet.
         """
-        runner = CliRunner()
+        os.mkdir("newsfragments")
+        with open("newsfragments/123.feature", "w") as f:
+            f.write("wow!\n~~~~\n\nNo indentation at all.")
+        with open("newsfragments/124.bugfix", "w") as f:
+            f.write("#. Numbered bullet list.")
+        with open("newsfragments/125.removal", "w") as f:
+            f.write("- Hyphen based bullet list.")
+        with open("newsfragments/126.doc", "w") as f:
+            f.write("* Asterisk based bullet list.")
 
-        with runner.isolated_filesystem():
-            with open("pyproject.toml", "w") as f:
-                f.write(
-                    "[tool.towncrier]\n"
-                    'template="towncrier:single-file-no-bullets"\n'
-                    "all_bullets=false"
-                )
-            os.mkdir("newsfragments")
-            with open("newsfragments/123.feature", "w") as f:
-                f.write("wow!\n~~~~\n\nNo indentation at all.")
-            with open("newsfragments/124.bugfix", "w") as f:
-                f.write("#. Numbered bullet list.")
-            with open("newsfragments/125.removal", "w") as f:
-                f.write("- Hyphen based bullet list.")
-            with open("newsfragments/126.doc", "w") as f:
-                f.write("* Asterisk based bullet list.")
-
-            result = runner.invoke(
-                _main,
-                [
-                    "--version",
-                    "7.8.9",
-                    "--name",
-                    "foo",
-                    "--date",
-                    "01-01-2001",
-                    "--yes",
-                ],
-            )
+        result = runner.invoke(
+            _main,
+            [
+                "--version",
+                "7.8.9",
+                "--name",
+                "foo",
+                "--date",
+                "01-01-2001",
+                "--yes",
+            ],
+        )
 
-            self.assertEqual(0, result.exit_code, result.output)
-            output = read("NEWS.rst")
+        self.assertEqual(0, result.exit_code, result.output)
+        output = read("NEWS.rst")
 
         self.assertEqual(
             output,
-            """
-foo 7.8.9 (01-01-2001)
-======================
+            dedent(
```

Comment:
just asking? Was this change needed? was the test failing with the old code?

For assertion, I prefer to have as little processing as possible.

But we can also have this code... no problem.

Reply:
Not specifically needed. But this way it aligns better with the other tests doing assertions the same way.

Reply:
Ok. We can merge this. No problem.